pkgs/top-level/all-packages.nix

```diff
@@ -8807,7 +8807,7 @@ with pkgs;
 
   peg = callPackage ../development/tools/parsing/peg { };
 
-  pgcli = callPackage ../development/tools/database/pgcli {};
+  pgcli = callPackage ../development/tools/database/pgcli { pythonPackages = python3Packages; };
```

Comment:
Please just use `python3Packages` within the derivation.

We'd like it to always appear that `python` proper is always python2.